root/programs/C#/Samples/WebApp_sample/ProjectX_sample/Scripts/touryo/common.js

```diff
@@ -603,81 +684,206 @@ function Fx_ShowMessageDialog(url) {
     url = url + "&Time=" + year_now + month_now + date_now + day_now + hour_now + minute_now + second_now;
 
     // マスクする。
-    Fx_MaskOn();
+    Fx_DialogMaskOn();
```

Comment:
@daisukenishino 

It might be a good idea to use `scrollWidth` method instead of `window.innerWidth` or `clientWidth`.

See also: [javascript jQuery スクロールバーの幅を含むウィンドウサイズを取得したい。](http://chaika.hatenablog.com/entry/2015/02/12/054602)